packages/paste-website/src/components/homepage/HomeHero.tsx

```diff
@@ -3,21 +3,44 @@ import {Anchor} from '@twilio-paste/anchor';
 import {Box} from '@twilio-paste/box';
 import {Grid, Column} from '@twilio-paste/grid';
 import {Text} from '@twilio-paste/text';
-import {NewComponentBanner} from './NewComponentBanner';
-import {NewComponentBannerBadge} from './NewComponentBannerBadge';
-import {NewComponentBannerLink} from './NewComponentBannerLink';
-import {NewComponentBannerText} from './NewComponentBannerText';
+import {ArrowForwardIcon} from '@twilio-paste/icons/esm/ArrowForwardIcon';
 import {HomeHeroIllustration} from './HomeHeroIllustration';
 import {SlantedBackgroundGradient} from '../SlantedBackgroundGradient';
 import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
 
+const SeeRoadmapAnchor: React.FC = () => {
+  const [hovered, setHovered] = React.useState(false);
+
+  const handleMouseEnter = (): void => {
+    setHovered(true);
+  };
+  const handleMouseLeave = (): void => {
+    setHovered(false);
+  };
+
+  return (
+    <Anchor href="/roadmap" variant="inverse" onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
```

Comment:
The focus state on this anchor is collapsing

![image](https://user-images.githubusercontent.com/368249/100795578-1f87aa00-33d4-11eb-9ada-d1e131f14af6.png)

I _think_ you can set display on the anchor, even though we don't advertise it.

Reply:
Verifying that you can set display on anchor. Shh. Don't tell anyone.